pkgs/by-name/ga/gambit-project/package.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv
+, fetchFromGitHub
+, autoreconfHook
+, wxGTK31
+, lib
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "gambit-project";
+  version = "16.1.1";
+
+  src = fetchFromGitHub {
+    owner = "gambitproject";
+    repo = "gambit";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-ElPzJDQ1q+i1OyliychSUA9pT6yGSwjn/sKV0JX5wrQ=";
+  };
+
+  nativeBuildInputs = [ autoreconfHook ];
+
+  buildInputs = [ wxGTK31 ];
+
```

Comment:
```suggestion

  strictDeps = true;

```

Reply:
This breaks wx-config autconf detection

Reply:
Even when `wxGTK31` is in both `nativeBuildInputs` and `buildInputs`?

If this is the case, maybe it should be included at `depsBuildBuild` too...

Reply:
Also, does this software use pkg-config?